010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,40 +1,29 @@
-- **Collaborative dashboard**: A web-based console to manage your data, including schema and data.
-- **SDKs**: Xata comes with vertically integrated SDKs for popular programming languages, including [TypeScript](/docs/sdk/typescript/overview), [JavaScript](/docs/sdk/typescript/overview), [Python](/docs/sdk/python/overview), and [Go](/docs/sdk/go/overview). In addition, Xata works with any [PostgreSQL client or ORM](/docs/postgres/).
-
-In order to provide these features, Xata uses other data stores in addition to PostgreSQL, namely [Elasticsearch](https://github.com/elastic/elasticsearch/) for full-text search, aggregations and vector search, [S3](https://aws.amazon.com/s3/) for file attachments, and the [Cloudflare CDN](https://www.cloudflare.com/application-services/products/cdn/) for file delivery and image transformations. The Postgres logical replication events are used to keep these data stores in sync with the main database.
-
-Access to the underlying data is provided in two main ways:
-
-- Via the PostgreSQL wire protocol, which is mediated by our SQL Proxy.
-- Via our SDKs, which provide a higher-level interface to the data.
-
-![PostgreSQL Data Platform](images/xata-platform.png)
+- **Location independent branches**: Branches can be easily moved from one Postgres cluster to another. This allows for simple **major version upgrades** as well for easy experimentation with various deployment options.
+- **[Full-text and vector search](/docs/sdk/search)**: Data is automatically replicated into a full-text search engine, making it easy to build search experiences, supporting fuzzy search, weights, and relevancy boosters.
+- **[File attachments](/docs/sdk/file-attachments)**: Store and retrieve files up to 5GB, including images, videos, and audio files. The files are stored in object storage and automatically cached on a global CDN for fast delivery. From the application point of view, however, it's just another column in the database. Any images stored with
+  File Attachments can perform powerful at-will image transformations.
 
 ## Who is Xata built for?
 
-Individuals and indie developers like Xata because it is easy to get started, has a generous free-tier, accelerates development speed and solves common problems in a developer-first way (e.g. file attachments, full-text search, vector search).
-
 Companies and larger teams like Xata because it is cost-efficient, consolidates tooling and reduces the operational load with its auto-scaling nature. Branching, and zero-downtime migration workflows means that your team can iterate faster when performing complex schema changes.
 
+Individuals and indie developers like Xata because it is easy to get started, has a generous free-tier, accelerates development speed and solves common problems in a developer-first way (e.g. file attachments, full-text search, vector search).
```

Comment:
```suggestion
Individuals and indie developers like Xata because it is easy to get started, has a generous free-tier, accelerates development speed and solves common problems in a developer-first way.
```

De-emphasize